src/hooks/useNotificationTypes.tsx

```diff
@@ -790,6 +792,60 @@ export const notificationTypes: NotificationTypeConfig[] = [
       };
     },
   },
+  {
+    type: NotificationType.PredictionMarketConcluded,
+    useTrigger: ({ trigger }) => {
+      const allFills = useAppSelector(getSubaccountFills, shallowEqual) ?? EMPTY_ARR;
```

Comment:
does this include ones filtered out after they're seen? Because we filter pretty aggressively after they are seen in the bottom panel.